en/docs/learn/api-controller/advanced-topics/creating-custom-users-to-perform-api-controller-operations.md

```diff
@@ -0,0 +1,148 @@
+#  Creating Custom Users to Perform API Controller Operations
+
+To perform tasks using the API Controller, a particular user must have required permissions and scopes. By default, a user with `admin` role is allowed perform all the tasks. 
+
+## Steps to Create a Custom User who can Perform API Controller Operations
+
+1. Create a role named `custom_role` with `API Create`, `API Publish` and `API Subscribe` permissions (along with `Login` permission) and assign it to a user.
+
+    !!! tip
+        Refer [Managing Users and Roles]({{base_path}}/administer/product-administration/managing-users-and-roles/introduction-to-user-management/) to learn how to create a user role with permissions and assign it to a user.
+
+2.  Assigning the required scopes for the `custom_role` can be done using 2 methods.
+
+    1.  Using API-M management console.
+
+        1. Sign in to the API-M management console (`https://<APIM_Host>:<APIM_Port>/carbon`) as a tenant admin user.
+
+        2. Navigate to **Main > Resources > Browse** 
+
+        3.  Enter `/_system/config/apimgt/applicationdata/tenant-conf.json` as the location and click **Go** to browse the registry and locate the required resource.
+
+        4.  Update the `RESTAPIScopes` JSON field by adding `custom_role` to the `Roles` field under the corresponding scope `Name` fields as shown below.
+            ```bash
+            {
+                "Name": "apim:api_view",
+                "Roles": "admin,Internal/publisher,Internal/creator,Internal/analytics,custom_role"
+            },
+            {
+                "Name": "apim:subscribe",
+                "Roles": "admin,Internal/subscriber,custom_role"
+            },
+            {
+                "Name": "apim:app_owner_change",
+                "Roles": "admin,custom_role"
+            },
+                "Name": "apim:app_import_export",
+                "Roles": "admin,custom_role"
+            },
+            {
+                "Name": "apim:api_import_export",
+                "Roles": "admin,custom_role"
+            },
+            ``` 
+
+    2.  Using API-M management console.
```

Comment:
The heding should be corrected as "Using API-M Admin Portal"

Reply:
Fixed via https://github.com/wso2/docs-apim/pull/2973